rules/windows/builtin/appxdeployment_server/appxdeployment_server_mal_appx_names.yml

```diff
@@ -8,14 +8,17 @@ references:
     - https://forensicitguy.github.io/analyzing-magnitude-magniber-appx/
 author: Nasreddine Bencherchali
 date: 2023/01/11
+modified: 2023/01/12
 tags:
     - attack.defense_evasion
 logsource:
     product: windows
     service: appxdeployment-server
 detection:
     selection:
-        EventID: 401
+        EventID:
+            - 400
+            - 401
```

Comment:
401 is fail  ,why not update the title and description ?

Reply:
Good idea. I will do that.